scheduler/src/cook/kubernetes/api.clj

```diff
@@ -537,10 +537,20 @@
                   (let [namespaced-pod-name {:namespace (.getNamespace involved-object)
                                              :name (.getName involved-object)}
                         host (some-> event .getSource .getHost)
-                        field-path (.getFieldPath involved-object)]
-                    (when (some-> @all-pods-atom
-                                  (get namespaced-pod-name)
-                                  (is-cook-scheduler-pod compute-cluster-name))
+                        field-path (.getFieldPath involved-object)
+                        event-type (.getType event)
+                        tracked-pod?
+                        (some-> @all-pods-atom
+                          (get namespaced-pod-name)
+                          (is-cook-scheduler-pod compute-cluster-name))]
+                    (when (or
+                            tracked-pod?
+                            (and
+                              ; The event type is either "Normal" or "Warning"
+                              (= event-type "Warning")
+                              ; The fieldPath is something like:
+                              ; "spec.containers{aux-cook-sidecar-container}"
+                              (some-> field-path str/lower-case (str/includes? "cook"))))
```

Comment:
is this the best we can do?

we can't identify something like

resource.labels.container_name="required-cook-job-container"

?

Reply:
the answer seems to be yes